### DISCUSSION TITLE
When the student benefits is going to be available and can I use the GitHub Co Pilot or not

### DISCUSSION THREAD

**Discussion by DivyanshVish** [community/community#60265] [GitHub Education]
### Hi everyone,

When the student benefits is going to be available and can I use the GitHub Co Pilot or not

**Comment by GhostOf0days** [comment]
Github Copilot is included in student benefits. Please wait 24 to 72 hours from the time of your approvals to reap the full benefits of your approval. Thank you.

**Comment by samus-aran** [comment]
Hi @DivyanshVish 👋 

Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
